packages/client/webapp/src/router/index.ts

```diff
@@ -4,8 +4,10 @@ import ResponsiveMatrixCells from '@/components/ResponsiveMatrixCells.vue';
 import TA2Playground from '@/views/TA2Playground.vue';
 import Theia from '@/views/theia.vue';
 
+export const HOME_PATH = '/';
```

Comment:
This should be nice as [enum](https://www.typescriptlang.org/docs/handbook/enums.html) of _ALL_ the routes.

Reply:
Done!